### ISSUE TITLE
Arch Linux package (AUR)

### ISSUE THREAD

**Issue by Trollwut** [jschmid1/gopro_as_webcam_on_linux#21]
Hey mates!

Any chance on getting an AUR package for this nice piece of software?

**Comment by NewoIsTaken** [comment]
Hey @Trollwut, I made my own copy of this project based on this code and put it on the AUR. It is right [here](https://aur.archlinux.org/packages/gopro-webcam/)!

**Comment by Trollwut** [comment]
Nice, thank you!!

The "Upstream URL" in the AUR site is broken btw :)

**Comment by NewoIsTaken** [comment]
@Trollwut, thanks for catching that! I think you can close this issue now.

**Comment by jschmid1** [comment]
> Hey @Trollwut, I made my own copy of this project based on this code and put it on the AUR. It is right [here](https://aur.archlinux.org/packages/gopro-webcam/)! 

Thanks for doing that after seeing these requests!

> @jschmid1, if you see this, please let me know if there are any changes you do not like or if you would like to be added as a maintainer.

Any reason you didn't fork the repo but rather created a new one?

**Comment by NewoIsTaken** [comment]
> Any reason you didn't fork the repo but rather created a new one?

Hi @jschmid1, the AUR requires that the repository that you push has the `.SRCINFO` file on every commit. I wasn't able to figure out how to put it onto the root commit so I just created a new repository so every commit would have the file on it.

**Comment by jschmid1** [comment]
ok, got it. if you like you can add my name/github handle as a maintainer. Happy to help. @NewoIsTaken

**Comment by NewoIsTaken** [comment]
> Keep in mind that you will have to manually sync changes to your repository when this repo gets updated
Yeah, that's something I'll have to deal with. Is it possible that you merge my changes? Let me open a pull request.

**Comment by jschmid1** [comment]
https://github.com/jschmid1/gopro_as_webcam_on_linux/pull/24 addressed here
